frontend/automation/pages/wallets.ts

```diff
@@ -4,14 +4,44 @@ import { expect } from '@playwright/test'
 const wallets = (page: Page) => {
   const locators = {
     goToCreateWalletButton: page.getByTestId('create-new-wallet'),
-    header: page.getByTestId('header-title')
+    goToImportWalleButton: page.getByTestId('import-wallet'),
+    header: page.getByTestId('header-title'),
+    passphraseForm: page.getByTestId('passphrase-form'),
+    passphraseField: page.getByTestId('input-passphrase'),
+    submitPassphraseButton: page.getByTestId('input-submit')
   }
-  return {
-    goToCreateWalletPage: async () => locators.goToCreateWalletButton.click(),
-    openWalletAndAssertName: async (walletName: string) => {
-      await page.getByTestId(`wallet-${walletName.replace(' ', '-')}`).click()
-      await expect(locators.header).toHaveText(walletName)
+
+  const goToCreateWalletPage = async () =>
```

Comment:
No need to cover one-liners with functions (applicable to all other similar functions)

Reply:
I am uncomfortable with this as a blanket statement to be honest